app/app.go

```diff
@@ -742,12 +742,39 @@ func New(
 		return app.mm.RunMigrations(ctx, app.configurator, fromVM)
 	})
 
+	gravityPlanName := "v0.8.0-gravity-alpha1"
+	app.UpgradeKeeper.SetUpgradeHandler(gravityPlanName, func(ctx sdk.Context, _ upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
```

Comment:
we can even put this logic under the condition `if experimental {`

Reply:
added